db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/Optimizable.java

```diff
@@ -428,4 +428,9 @@ boolean feasibleJoinStrategy(OptimizablePredicateList predList,
 	 * @exception StandardException		Thrown on error
 	 */
 	double uniqueJoin(OptimizablePredicateList predList) throws StandardException;
+
+	/**
+	 * get the current optimizable's memory usage if its best join plan is a broadcast join
+	 */
+	double getMemoryUsage4BCJ();
```

Comment:
I'd rather have a complete name, rather than the abbreviature. I wouldn't mind on a local variable but for the method name I think it's better to spell it out.